.github/ISSUE_TEMPLATE/bug_report_cli.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "🐛 Bug Report in Expo CLI"
+about: You want to report an issue with Expo CLI, our command line tool.
+---
+
+<!--
+  This is not the repository for Expo CLI. Please post this issue in the https://github.com/expo/expo-cli repo instead.
+-->
+
+### Please note that Expo CLI issues opened in the core Expo repository will be closed.
```

Comment:
Should these notes be uncommented? Maybe they should be commented in their own comment, upcased?

Reply:
The advantage of not commenting this part is that everyone will see it when someone submitted such issue. Also, that's how RN does 😅